<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureFaceMg</class>
 <widget class="QWidget" name="PictureFaceMg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PictureFaceMg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#fff</string>
  </property>
  <widget class="QLabel" name="label_input1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #169fe6;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_input2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #169fe6;
border-radius: 6px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_input1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
background-color: #169fe6;
padding: 11px 19px;
font-size: 17px;
border-radius: 6px;
font-family: &quot;Microsoft YaHei UI&quot;</string>
   </property>
   <property name="text">
    <string>选择上图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_input2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
background-color: #169fe6;
padding: 11px 19px;
font-size: 17px;
border-radius: 6px;
font-family: &quot;Microsoft YaHei UI&quot;</string>
   </property>
   <property name="text">
    <string>选择下图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_faceSwap">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
background-color: #169fe6;
padding: 11px 19px;
font-size: 17px;
border-radius: 6px;
font-family: &quot;Microsoft YaHei UI&quot;</string>
   </property>
   <property name="text">
    <string>进行换脸</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
background-color: #169fe6;
padding: 11px 19px;
font-size: 17px;
border-radius: 6px;
font-family: &quot;Microsoft YaHei UI&quot;</string>
   </property>
   <property name="text">
    <string>保存图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_similarity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
background-color: #169fe6;
padding: 11px 19px;
font-size: 17px;
border-radius: 6px;
font-family: &quot;Microsoft YaHei UI&quot;</string>
   </property>
   <property name="text">
    <string>相似度检测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_similarity">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>850</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;
text-shadow: 0 -1px 0 rgba(0,0,0,0.25);
background-color: #169fe6;
font-size: 17px;
border-radius: 6px;
font-family: &quot;Microsoft YaHei UI&quot;</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_similarity_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px
</string>
   </property>
   <property name="text">
    <string>相似度：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_input1</sender>
   <signal>clicked()</signal>
   <receiver>PictureFaceMg</receiver>
   <slot>inputImage1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_input2</sender>
   <signal>clicked()</signal>
   <receiver>PictureFaceMg</receiver>
   <slot>inputImage2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_faceSwap</sender>
   <signal>clicked()</signal>
   <receiver>PictureFaceMg</receiver>
   <slot>faceSwap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_similarity</sender>
   <signal>clicked()</signal>
   <receiver>PictureFaceMg</receiver>
   <slot>similarityDetection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>PictureFaceMg</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_back</sender>
   <signal>clicked()</signal>
   <receiver>PictureFaceMg</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>871</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>867</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputImage1()</slot>
  <slot>inputImage2()</slot>
  <slot>faceSwap()</slot>
  <slot>similarityDetection()</slot>
  <slot>back()</slot>
  <slot>save()</slot>
 </slots>
</ui>
